source/develop/release-management/features/infra/engine-snmp3.md

```diff
@@ -91,6 +91,10 @@ Edit /etc/snmp/snmptrapd.conf to add the library specific directive in front of
       # Log incoming traps to /var/log/snmptrapd.log
       [snmp] logOption f /var/log/snmptrapd.log
 
+You will have to change SELinux settings on /var/log/snmptrapd.log in order to get write permissions
+
+chcon -t snmpd_log_t /var/log/snmptrapd.log
```

Comment:
Isn't this a selinux policy bug? Did you report it?

Reply:
1. The snmptrapd configuration is not documented in oVirt

2. I don't think its a SELinux policy bug , the file is defined in /etc/snmptrapd.conf :

[snmp] logOption f /var/log/snmptrapd.log

It is the responsibility of the admin to set correctly SELinux type IMO